File: Curso_de_Programacion_Orientada_a_Objetos-POO/Java/UberX.java
class UberX extends Car {
  String brand;
  String model;

  public UberX(String license, Account driver, String brand, String model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;
  }

  void printDataUberX() {
    System.out.println(
        "Car licence: " + this.license + " Driver: " + this.driver.name + " Documento: " + this.driver.document
            + " Pasajeros: " + this.passenger + " Marca: " + this.brand + " Modelo: " + this.model);

  }

}

File: Curso_de_Programacion_Orientada_a_Objetos-POO/JS/UberX.js
class UberX extends Car {
	constructor(license, driver, brand, model) {
		super(license, driver);
		this.brand = brand;
		this.model = model;
	}

	printDataUberX() {
		console.log(
			"Car licence: " +
				this.license +
				" Driver: " +
				this.driver.name +
				" Documento: " +
				this.driver.document +
				" Marca: " +
				this.brand +
				" Modelo: " +
				this.model
		);
	}
}
